Dashboard Cleanup

murrant opened the pull request:
Fix some html issues
Remove spacing in availability map
Add icons to availability map
Ajax setup has already been called in the layout with the proper token.

murrant committed:
Dashboard cleanup
Fix some html issues
Remove spacing in availabilty map
Add icons to availibility map

murrant committed:
Ajax setup has already been called in the layout with the proper token.

laf commented on resources/views/home.blade.php:
```diff
@@ -74,7 +74,6 @@
                         <div class="form-group">
                             {{ Form::label('copy_from', trans('dashboard.label.copy')) }}
                             <select class="form-control" id="copy_from" name="copy_from">
-                                <option></option>
```
You don't necessarily want to copy from another dashboard so you need the blank option.

laf commented on resources/views/home.blade.php:
```diff
@@ -93,7 +92,6 @@
                     <div class="form-group">
                         {{ Form::label('name', trans('dashboard.label.add_widget')) }}
                         <select class="form-control" id="add_widget_id" name="add_widget_id" data-dashboard_id="{{ $request->route('dashboard_id') }}">
-                                <option value=""></option>
```
I've left this in as if you want to add two of the same widget you would have to refresh the page.

murrant committed:
Re-add blank options
spacing to the right of widgets
draggable cursor
re-order css

murrant committed:
Seperate out draggable and padding

murrant committed:
Remove a little padding in some places so the widgets fit better without scrollbars.
Remove the Unused WidgetsContoller.php
Optimize some queries in teh WidgetDataController.

murrant committed:
Remove some unused code
Work on simplifying things.
Add crosfade on config show.
Clicking wrench closes settings again.
Use class instead of id for things that might have multiples.
Update phpdocs

murrant committed:
Unused code removal.

murrant committed:
Merge remote-tracking branch 'refs/remotes/origin/fixes' into fixes

murrant committed:
Fix tests.
More files removed.
More phpdoc updates.
